leshan-server-redis/src/main/java/org/eclipse/leshan/server/redis/RedisSecurityStore.java

```diff
@@ -167,4 +180,90 @@ public void addListener(SecurityStoreListener listener) {
     public void removeListener(SecurityStoreListener listener) {
         listeners.remove(listener);
     }
+
+    /**
+     * Class helping to build and configure a {@link RedisSecurityStore}.
+     * <p>
+     * By default, uses {@code SECSTORE#} prefix for all keys, {@code SEC#EP#} key prefix to find security info by
+     * endpoint and {@code EP#PSKID} key to get the endpoint by PSK ID. Leshan v1.x used {@code SEC#EP#} and
+     * {@code PSKID#SEC} keys for that accordingly.
+     */
+    public static class Builder {
+        private Pool<Jedis> pool;
+        private String securityInfoByEndpointPrefix;
+
+        private String endpointByPskIdKey;
+
+        private String prefix;
+
+        /**
+         * Set the Redis connection pool for the {@link RedisSecurityStore}.
+         */
+        public void setPool(Pool<Jedis> pool) {
+            this.pool = pool;
+        }
```

Comment:
As we give pool as Builder constructor maybe we don't need this setter or do you have a use case in mind ?

Reply:
Good point, I removed this setter.